Doxyfile.in

```diff
@@ -514,7 +551,7 @@ INTERNAL_DOCS          = NO
 # names in lower-case letters. If set to YES, upper-case letters are also
 # allowed. This is useful if you have classes or files whose names only differ
 # in case and if your file system supports case sensitive file names. Windows
-# and Mac users are advised to set this option to NO.
+# (including Cygwin) ands Mac users are advised to set this option to NO.
```

Comment:
and~s~  :D

Reply:
that was automatically written by doygen, using `doxygen -u Doxyfile.in`, so the issue is on their side :smile:

Reply:
:joy: didn't thought about where it's coming from, but it's indeed in their [current master](https://github.com/doxygen/doxygen/blob/e8f268edd2e440c136735e878e4c9d79d57354ae/src/config.xml#L963). This just jumped at me while scrolling through.